test/SPIRV/ops.mlir

```diff
@@ -1,5 +1,123 @@
 // RUN: mlir-opt -split-input-file -verify-diagnostics %s | FileCheck %s
 
+//===----------------------------------------------------------------------===//
+// spv.AccessChain
+//===----------------------------------------------------------------------===//
+
+func @access_chain_struct() -> () {
+  %0 = spv.constant 1: index
+  %1 = spv.Variable : !spv.ptr<!spv.struct<f32, !spv.array<4xf32>>, Function>
+  // CHECK: %2 = spv.AccessChain %1[%0, %0] : !spv.ptr<!spv.struct<f32, !spv.array<4 x f32>>, Function>
```

Comment:
Do not check the names of the SSA values directly, use regex instead:
(See https://github.com/tensorflow/mlir/blob/master/g3doc/TestingGuide.md)

Reply:
Thanks, fixed.